docs/releases/1.20-NOTES.md

```diff
@@ -50,6 +50,8 @@
 
 * Due to lack of maintainers, the Aliyun/Alibaba Cloud support has been deprecated. The current implementation will be left as-is until the implementation needs updates or otherwise becomes incompatible. At that point, it will be removed. We very much welcome anyone willing to contribute to this cloud provider.
 
+* Support for AWS LaunchConfiguration is being deprecated and will be removed in kOps 1.21.
```

Comment:
```suggestion
* Support for AWS LaunchConfiguration has been deprecated and will be removed in kOps 1.21.
```

Reply:
Thanks! Will make these changes and push!